### DISCUSSION TITLE
Ground Station can not open terminal.

### DISCUSSION THREAD

**Discussion by mmucahitkaya** [svpcom/wfb-ng#219] [Questions]
Hi, i setup image on my rpi3. I can stream from my drone to ground station but i can't close OSD in ground station and i can't access terminal. How can I close it and how can i access the terminal in GS? 

![image](https://user-images.githubusercontent.com/39586436/169605917-196074ba-c74a-4c05-983d-53d71e1d07b7.png)

**Comment by svpcom** [comment]
ssh to the gs and stop osd service via `systemctl stop`

**Reply by mmucahitkaya** [reply]
I can't connect ssh in my pc. I connect GS to my pc via ethernet.  How can i find GS ip adress? I know that's question is very easy maybe but i'm a noob :(

**Reply by svpcom** [reply]
Connect GS to your home router via ethernet (or configure dhcp server on your pc if no router is available). Then see router logs to found what ip address was issued to the GS
